README.md

```diff
@@ -26,6 +26,13 @@ Make sure you have following required packages installed:
 * `curl`
 * `gpg2`
 
+Fetch our GPG keys:
+
+`gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB`
+
+In case you run into troubles use different key server, for example:
```

Comment:
It reads a better to say "use a different" here.

Reply:
Sure! Fixed!